README.md

```
@@ -1,68 +1,147 @@
+/>
+```
+
+### Themed Map
+```tsx
+// Automatically adapts to your app's light/dark theme
+<Map theme="auto" />
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)